scripts/pagespeed_libraries_generator.sh

```diff
@@ -27,9 +27,9 @@
 # Author: vid@zippykid.com (Vid Luther)
 #         jefftk@google.com (Jeff Kaufman)
 
-URL="https://github.com/pagespeed/mod_pagespeed/raw/master/"
-URL+="net/instaweb/genfiles/conf/pagespeed_libraries.conf"
-curl -L -s -S "$URL" \
+URL="https://raw.githubusercontent.com/pagespeed/mod_pagespeed/master/"
+URL2="net/instaweb/genfiles/conf/pagespeed_libraries.conf"
```

Comment:
I thought that the change from `https://github.com` to `https://raw.githubusercontent.com` is what fixed the problem? I think that that is all that needs to be changed.